### DISCUSSION TITLE
How to know SMS is received from mobile phone to server?

### DISCUSSION THREAD

**Discussion by Woyram** [python-smpplib/python-smpplib#213] [Q&A] [answered]
Hi @eigenein, @podshumok 

I am implementing python `smpplib` to make custom SMS gateway.

I am successfully able to send SMS by using the code in `readme.md` file.

I go read `client.py` code in library. I see there is `def set_message_received_handler(self, func)` method
it is triggered after I sent SMS from script. But it is not triggered when is send SMS from mobile phone 
to that script running server.

Could you please help to figure out how to confirm the SMS is received in script running server when I send SMS from 
phone?

**Comment by eigenein** [comment, marked as answer]
Duplicate of #164.
